stdlib/release/src/commons/functions.bal

```diff
@@ -91,14 +95,15 @@ public function logAndPanicError(string message, error e) {
 }
 
 public function printModules(Module[] modules) {
-    string[] moduleStrings = modules.map(function (Module m) returns string {
-        return m.name + " " + m.'version;
-    });
-    foreach string moduleString in moduleStrings {
-        log:printInfo(moduleString);
+    foreach Module module in modules {
+        printModule(module);
     }
 }
 
+public function printModule (Module module) {
+    log:printInfo(module.name + " " + module.'version);
+}
+
```

Comment:
Any reason to have a separate function? It's just a log isnt it?

Reply:
Just for the convenience. To use in any place with the module name and the version.

Reply:
Ack